pkgs/servers/nextcloud/packages/29.json

```diff
@@ -90,9 +90,9 @@
     ]
   },
   "forms": {
-    "hash": "sha256-J2xGGZ4fEUwpWeEi3swdr9bwl4iBgltHGxo9Bi+Tdus=",
-    "url": "https://github.com/nextcloud-releases/forms/releases/download/v4.3.5/forms-v4.3.5.tar.gz",
-    "version": "4.3.5",
+    "hash": "sha256-Fso4vxcn+4lCuWpgPMmp4dUMMeDaZrXne3CCnt4p7Rg=",
+    "url": "https://github.com/nextcloud-releases/forms/releases/download/v4.3.6/forms-v4.3.6.tar.gz",
```

Comment:
This change is in the wrong commit.

Reply:
What do you mean with wrong here specifically?

Reply:
It's part of the commit moving the notify_push app rather than the one updating all apps.